### PR TITLE
misc cleanups to all CI platforms

### PR DESCRIPTION
**Pull request by maaktweluit**

### CODE REVIEW

File: shippable.yml [hunk 1]
```diff
@@ -4,13 +4,8 @@ python:
   - 3.5
 
 install:
-  - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
-  - sudo add-apt-repository -y ppa:ethereum/ethereum && sudo apt-get update
-  - sudo apt-get install -y qt5-default qtbase5-dev-tools geth unzip gcc-5 g++-5 libfreeimage3 libfreeimage-dev
-  - pip3 install six
   - pip3 install -r requirements.txt
   - pip3 install -r requirements-lint.txt
-  - make -C apps/rendering/resources/taskcollector
```
Comment (etam) [inline shippable.yml:13]: I think that building taskcollector should be tested (it requires libfreeimage-dev)

**Review by etam — CHANGES_REQUESTED** [review]

File: shippable.yml (hunk 1 above)
Comment (Elfoniok) [reply shippable.yml:13]: Hmm shipable probably just don't use taskcollector, so it can be skipped?

File: circle.yml [hunk 2]
```diff
@@ -44,14 +45,10 @@ dependencies:
     - /usr/local/bin/ipfs daemon:
         background: true
```
Comment (etam) [inline circle.yml:46]: Is ipfs still needed? AFAIK we're not using it anymore, so this could be deleted

Comment (maaktweluit) [reply circle.yml:46]: nope, no longer needed, but removed in #1620 ( also up for review and merge :) )

File: shippable.yml (hunk 1 above)
Comment (maaktweluit) [reply shippable.yml:13]: i think linters do not use it at all, the light is green on the PR so "assumed" it was good like this.

**Review by Elfoniok — APPROVED** [review]

File: shippable.yml (hunk 1 above)
Comment (etam) [reply shippable.yml:13]: Do we have any tests for taskcollector? If not at least we could test if it builds.

Comment (etam) [reply shippable.yml:13]: Well, on shippable we only run linters, not full tests, so this can be dropped from here.

**Review by etam — APPROVED** [review]

**Pull request merged**